### ISSUE TITLE
small loss but bad inference

### ISSUE THREAD

**Issue by elesun2018** [zylo117/Yet-Another-EfficientDet-Pytorch#224]
When i train on VOC2012 dataset , loss continuous decline,(min ~ 0 ), but for inference bad results.
![image](https://user-images.githubusercontent.com/37207093/80510945-033f0d00-89ae-11ea-8421-ba63f3af6a83.png)

thanks.

**Comment by elesun2018** [comment]
![图片1](https://user-images.githubusercontent.com/37207093/80511511-c6274a80-89ae-11ea-9b47-18ad1ceb3ef1.png)
![Uploading 图片2.png…]()

**Comment by zylo117** [comment]
what matters is val loss. it should be around the training loss

**Comment by elesun2018** [comment]
thanks，loss in the pic is training loss.
logs as follows:

Step: 23999. Epoch: 52/60. Iteration: 28/461. Cls loss: 0.03494. Reg loss: 0.01194. Total loss: 0.04688:   6%|▌         | 27/461 [00:23<04:48,  1.51it/s]save checkpoint...
Step: 24432. Epoch: 52/60. Iteration: 461/461. Cls loss: 0.02108. Reg loss: 0.01492. Total loss: 0.03600: 100%|██████████| 461/461 [05:08<00:00,  1.50it/s]
Val. Epoch: 52/60. Classification loss: 0.09998. Regression loss: 0.02261. Total loss: 0.12260
Step: 24499. Epoch: 53/60. Iteration: 67/461. Cls loss: 0.02186. Reg loss: 0.01562. Total loss: 0.03749:  15%|█▍        | 67/461 [00:47<04:27,  1.47it/s]save checkpoint...
Step: 24893. Epoch: 53/60. Iteration: 461/461. Cls loss: 0.00477. Reg loss: 0.00563. Total loss: 0.01040: 100%|██████████| 461/461 [05:06<00:00,  1.50it/s]
Val. Epoch: 53/60. Classification loss: 0.08920. Regression loss: 0.01894. Total loss: 0.10814

val total loss ~0.1, train total loss 0.01~0.03

**Comment by zylo117** [comment]
isn't that totally overfitting? val loss is 10 times bigger than training loss

**Comment by elesun2018** [comment]
if i train the coco dataset from scratch，What kind of training loss can be accepted ?
training loss 1 or 0.1
thanks.

**Comment by elesun2018** [comment]
OK, then, how can I fit this problem(totally overfitting) ? thanks.

**Comment by zylo117** [comment]
val loss
cls 0.25
reg 0.025

**Comment by xiaoshuyun** [comment]
> When i train on VOC2012 dataset , loss continuous decline,(min ~ 0 ), but for inference bad results.
> ![image](https://user-images.githubusercontent.com/37207093/80510945-033f0d00-89ae-11ea-8421-ba63f3af6a83.png)
> 
> thanks.

hello.
I want to know how to train on VOC dataset?
